views/applications/_form.html

```diff
@@ -0,0 +1,13 @@
+<form action="{{action}}" method="post" enctype="multipart/form-data" class="form" novalidate>
```

Comment:
Why this over the previously created [basic form](https://github.com/punchcard-cms/punchcard/blob/master/views/_basic-form.html)